### DISCUSSION TITLE
VERO WIKI

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#3] [General]
# VERO WIKI

개발 [ gatsby ] [ programming-book|개발 책 ] [ short-programming-knowledge|짧은 개발 지식 ] TIL [ 2023 ] [ wooteco|우테코 ]

https://cyma-s.github.io/

**Comment by theminjunchoi** [comment]
> 누군가가 내 위키를 보고 자신만의 위키를 만들게 된다면

저 입니다 👍🏻
